examples/notebooks/beam-ml/bigtable_enrichment_transform.ipynb

```diff
@@ -509,15 +509,48 @@
         "\n",
-        "To convert a `string` type to a `byte` type or a `byte` type to a `string` type from Bigtable, you can configure additional options, such as [`app_profile_id`](https://cloud.google.com/bigtable/docs/app-profiles), [`row_filter`](https://cloud.google.com/python/docs/reference/bigtable/latest/row-filters), and [`encoding`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.bigtable.html#apache_beam.transforms.enrichment_handlers.bigtable.BigTableEnrichmentHandler:~:text=for%20more%20details.-,encoding,-(str)%20%E2%80%93%20encoding) type.\n",
+        "Optionally, to provide additional configuration to the `BigTableEnrichmentHandler` handler, see [module docs](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.bigtable.html#module-apache_beam.transforms.enrichment_handlers.bigtable)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yFMcaf8i7TbI"
+      },
+      "source": [
+        "**Note:** When exceptions occur, by default, the logging severity is set to warning ([`ExceptionLevel.WARN`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.utils.html#apache_beam.transforms.enrichment_handlers.utils.ExceptionLevel.WARN)).  To configure the severity to raise exceptions, set `exception_level` to [`ExceptionLevel.RAISE`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.utils.html#apache_beam.transforms.enrichment_handlers.utils.ExceptionLevel.RAISE). To ignore exceptions, set `exception_level` to [`ExceptionLevel.QUIET`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.enrichment_handlers.utils.html#apache_beam.transforms.enrichment_handlers.utils.ExceptionLevel.QUIET).\n",
         "\n",
-        "The default `encoding` type is `utf-8`.\n",
+        "The following example demonstrates how to set exception level in `BigTableEnrichmentHandler`:\n",
         "\n",
-        "\n"
+        "```\n",
+        "bigtable_handler = BigTableEnrichmentHandler(project_id=PROJECT_ID,\n",
+        "                                             instance_id=INSTANCE_ID,\n",
+        "                                             table_id=TABLE_ID,\n",
+        "                                             row_key=row_key,\n",
+        "                                             exception_level=ExceptionLevel.RAISE)\n",
+        "```"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The key component of the Bigtable handler is the `row_key` parameter. The `row_key` represent the field in input schema (`beam.Row`) that contains the row key for a row in the table.\n",
+        "\n",
+        "As of Apache Beam version 2.54.0, if the table uses composite row keys, then you can:\n",
```

Comment:
```suggestion
        "Starting in Apache Beam version 2.54.0, if the table uses composite row keys, then you can do the following tasks:\n",
```

Reply:
I've phrased this slightly different because users only need to do one of these steps